e2e/testsuite/testsuite.go

```diff
@@ -484,6 +491,39 @@ func (s *E2ETestSuite) ExecuteGovProposal(ctx context.Context, chain *cosmos.Cos
 	s.Require().Equal(govtypes.StatusPassed, proposal.Status)
 }
 
+
+// QueryModuleAccountAddress returns the sdk.AccAddress of a given module name.
+func (s *E2ETestSuite) QueryModuleAccountAddress(ctx context.Context, moduleName string, chain *cosmos.CosmosChain) (sdk.AccAddress, error) {
+	authClient := s.GetChainGRCPClients(chain).AuthQueryClient
+
+	moduleAccountsResponse, err := authClient.ModuleAccounts(ctx, &authtypes.QueryModuleAccountsRequest{})
+	if err != nil {
+		return nil, err
+	}
+
+	cfg := simappparams.MakeTestEncodingConfig()
+	authtypes.RegisterInterfaces(cfg.InterfaceRegistry)
+
+	for _, acc := range moduleAccountsResponse.Accounts {
+		var account authtypes.AccountI
+		err := cfg.InterfaceRegistry.UnpackAny(acc, &account)
+		if err != nil {
+			return nil, err
+		}
+		moduleAccount, ok := account.(authtypes.ModuleAccountI)
+		if !ok {
+			return nil, errors.New(fmt.Sprintf("failed to cast account: %T as ModuleAccount", moduleAccount))
```

Comment:
Just curious on your usage of `errors.New(fmt.Sprintf())` vs `fmt.Errorf()`?

Reply:
this was just personal preference, although now I see that we use `fmt.Errof` in other places in this file, so am included to change it to keep consistent.

Reply:
whatever you think is best :+1: